.flowconfig

```diff
@@ -7,6 +7,9 @@
 # Ignore compiled code.
 .*/lib/.*
 
+# Ignore node_modules
+.*/node_modules/*
+
```

Comment:
I do not think it's a good idea to ignore node_modules. We should instead create stubs for any node module that has issues/errors.

Reply:
Yeah, because when i tried to do `flow` there's some package in our `node_modules` that throws error, should i just ignore some of the problematic ones ?